samples/extensions/AWSSQS/connector/sqs/connector.json

```diff
@@ -45,8 +46,8 @@
       "type": "string",
       "required": true,
       "display": {
-        "name": "Secret Access Key",
-        "description": "AWS Secret Access Key for the user",
+        "name": "Secrete Access Key",
```

Comment:
Should be **Secret**

Reply:
have made the changes and also have made the connector.ts file changes.